cmd/teleirc.go

```diff
@@ -10,37 +10,46 @@ import (
 	tg "github.com/ritlug/teleirc/internal/handlers/telegram"
 )
 
-func startTelegram() {
-	tgbot, err := tg.StartBot()
-	if err != nil {
-		fmt.Println(err)
-	}
+var (
+	version = "v2.0"
 
-	fmt.Println(tgbot)
-}
+	flagPath    = flag.String("conf", ".env", "config file")
+	flagDebug   = flag.Bool("debug", false, "enable debugging")
+	flagVersion = flag.Bool("version", false, "displays current version of TeleIRC")
+)
 
 func main() {
-	// optional path flag if user does not want .env file in root dir
-	var path string
-	flag.StringVar(&path, "p", ".env", "Path to .env")
-
 	flag.Parse()
 
-	settings, err := internal.LoadConfig(path)
+	if *flagVersion {
+		fmt.Printf("Current TeleIRC version: %s\n", version)
+		return
+	}
+
+	// TODO: Build out debugging capabilities for more verbose output
+	if *flagDebug {
+		fmt.Printf("Debug mode currently set to: %s\n", flagDebug)
```

Comment:
`go-vet` complains about this being `%s` instead of `%t`, and I believe `flagDebug` should be `*flagDebug`.